### DISCUSSION TITLE
Active website is not displaying properly

### DISCUSSION THREAD

**Discussion by RebeccaCrecelius** [community/community#122426] [Programming Help]
### Launched Website Problems

Question

### Hamburger Menu Not Working & Styling Issues

Recently, I launched a repository as an active website. The hamburger menu on the launched website does not work. This issue is not due to a connectivity with the JavaScript file, because I also have a JavaScript feature on the homepage. It is working fine. 

I also checked the local code again. It works fine. I updated the repository with the code which was verified as working, and the hamburger menu still does not work on the launched website. 

Additionally, when viewed on mobile, buttons have styling which is not listed on the CSS. In fact, they show a completely color than what is listed. This is also not an issue when viewing the local version. 

Does anyone have any ideas on what is causing the problem?

**Comment by tstran155** [comment]
Hi RebeccaCrecelius,

There are a few things you can try:

- Clear your browser's cache or try accessing the site using a different browser or private/incognito mode or disable caching in your browser's developer tools settings.
- It could be from the CSS overriding and media queries in your CSS might apply different styles based on the viewport size. 
- The issue could be from the hosting environment's configuration like directories are not accessible or if there's an issue with how static files are served.

Hope it helps!

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @RebeccaCrecelius ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
